### PR TITLE
Version 3

### PR DESCRIPTION
**Pull request by rossabaker**
* Add MetadataPlugin
* Add DocsPlugin
* Demote ScalaCheck and ScalaTest to optional plugins
* Turn on coverage by default in Travis

### CODE REVIEW

File: src/main/scala/DocsPlugin.scala [hunk 1]
```diff
@@ -0,0 +1,117 @@
+package verizon.build
```
Comment (timperrett) [inline src/main/scala/DocsPlugin.scala:17]: @rossabaker This file requires a license header.

File: src/main/scala/DocsPlugin.scala [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  import scoverage.ScoverageKeys.coverageEnabled
+
+  /**
+   * These settings should only ever be used in isolation, on a documentation
+   * specific module of your project.
+   */
+  def settings =
+    Seq(
+      coverageEnabled := false,
+      gitRemoteHost := "github.com",
```
Comment (timperrett) [inline src/main/scala/DocsPlugin.scala:73]: Isn't this the default?

**Review by timperrett — CHANGES_REQUESTED** [review]
@rossabaker overall looks good, just those couple of comments :-)

File: src/main/scala/DocsPlugin.scala (hunk 2 above)
Comment (rossabaker) [reply src/main/scala/DocsPlugin.scala:73]: Yes.  That key is defined by this plugin, so this is where we're making the the default.

**Review by timperrett — APPROVED** [review]

**Pull request merged**